packages/react-sdk/src/components/ImportWhiteboardDialog/ImportWhiteboardDialog.tsx

```diff
@@ -172,11 +252,120 @@ export function ImportWhiteboardDialog({
         <Button
           onClick={handleOnImport}
           variant="contained"
-          disabled={importedWhiteboard?.isError}
+          disabled={error || loading}
         >
           {t('boardBar.importWhiteboardDialog.success.import', 'Import')}
         </Button>
       </DialogActions>
     </Dialog>
   );
 }
+
+function readNWB(file: File): Promise<{
```

Comment:
Also the read functions could use some comment about what they do and how they do or don't handle errors.